src/network/netdev/xfrm.c

```diff
@@ -0,0 +1,71 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <net/if.h>
+
+#include "netdev/xfrm.h"
+
+static int xfrm_config_verify(NetDev *netdev, const char *filename) {
+        Xfrm *x;
+
+        assert(netdev);
+        assert(filename);
+
+        x = XFRM(netdev);
+
+        if (!x->dev) {
+                log_warning("Xfrm NetDev without dev set in %s. Defaulting to 'lo'", filename);
+                if(!((x->dev = strdup("lo")))) {
+                        log_error("Could not allocate memory");
+                        return -ENOMEM;
```

Comment:
have you seen my earlier review? pleas use log_oom() for oom errors. Also please don't do the `x = strdup("lo")` inside an `if` expression: please do two things in two lines, don't compress everything into a single one